content/docs/hooks-effect.md

```diff
@@ -203,13 +203,7 @@ You might be thinking that we'd need a separate effect to perform the cleanup. B
 import React, { useState, useEffect } from 'react';
 
 function FriendStatus(props) {
-  const [isOnline, setIsOnline] = useState(null);
-
   useEffect(() => {
-    function handleStatusChange(status) {
```

Comment:
This functions is needed as the callback below. 

```js
ChatAPI.subscribeToFriendStatus(props.friend.id, handleStatusChange); 
```

The example is showing why it's important to clean up after effects; otherwise, this `handleStatusChange` will subscription will keep being called after the component is unmounted or when the friend id changes

Reply:
@alexkrolick looks like `setIsOnline` is sufficient in this way, what's the point of separate function in `useEffect`? And `isOnline` has a boolean naming for the string value. Let me know if you want me to change it to the `status`.